### DISCUSSION TITLE
Cannot block gif image.

### DISCUSSION THREAD

**Discussion by hoanganclub** [AdguardTeam/AdGuardHome#3625] [Q&A]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.106.3
* **How did you install AdGuard Home:**
  * GitHub  script on readme
* **How did you setup DNS configuration:**
  * IoT
* **If it's a router or IoT, please write device model:**
  * OrangePi One
* **CPU architecture:**
  * Armbian
* **Operating system and version:**
  * Armbian Focal(mainline based kernel 5.10.y)

### Expected Behavior
Website shouldn't have gif images
I set both custom filter and blocklist.

### Actual Behavior
Website still load gif image

### Screenshots

<details><summary>Screenshot:</summary>

</details>

![ảnh](https://user-images.githubusercontent.com/37793829/133915964-3c843e6c-02db-44eb-8ee7-6c74111563e4.png)

![ảnh](https://user-images.githubusercontent.com/37793829/133915936-aadc6ad7-3f7e-4efe-b9ec-1ee6bdedb669.png)

![ảnh](https://user-images.githubusercontent.com/37793829/133915919-eaa6649a-cd45-43c0-af49-1024dcc08ea3.png)

**Comment by schorschfunke** [comment]
Afaik, you can only block whole domains and not the content of a website. This is not a content blocker, it is a host blocker!

**Comment by Aikatsui** [comment]
@hoanganclub you have to wait for https://github.com/AdguardTeam/AdGuardHome/issues/1228 until you can use adguard extensions
